Bio/Data/PDBData.py

```diff
@@ -290,3 +290,99 @@
     "45A": "A", "4U3": "C", "8AG": "G", "8AN": "A", "8BA": "A", "8FG": "G",
     "8MG": "G", "8OG": "G", "8PY": "G", "8AA": "G", "85Y": "U", "8OS": "G",
 }
+
+# Solvent accesibility scales
+residue_max_acc = {
+    # Miller max acc: Miller et al. 1987 https://doi.org/10.1016/0022-2836(87)90038-6
```

Comment:
Being super nitpicky about format, I'd sort these alphabetically and put the reference before each one.